src/components/People/index.tsx

```diff
@@ -93,6 +109,56 @@ export default function PeopleList({ organization, reloadData, user }: PeopleLis
     }
   }, [organization]);
 
+  useEffect(() => {
+    const refreshSearch = async () => {
+      if (debouncedSearchTerm) {
+        const params: SearchNodePayload = {
+          prefix: 'members',
+          fields: ['user_id', 'user_firstName', 'user_lastName', 'user_email', 'roleName', 'createdTime'],
+          search: {
+            operation: 'and',
+            children: [
+              {
+                operation: 'or',
+                children: [
+                  { operation: 'field', field: 'user_firstName', type: 'Fuzzy', values: [debouncedSearchTerm] },
+                  { operation: 'field', field: 'user_lastName', type: 'Fuzzy', values: [debouncedSearchTerm] },
+                ],
+              },
+              {
+                operation: 'field',
+                field: 'organization_id',
+                type: 'Exact',
+                values: [organization?.id],
+              },
+            ],
+          },
+          count: 0,
+        };
+        const requestId = Math.random().toString();
+        setRequestId('searchUsers', requestId);
+        const searchResults = await search(params);
+        const usersResults: OrganizationUser[] = [];
+        searchResults?.forEach((result) => {
+          usersResults.push({
+            firstName: result.user_firstName as string,
+            lastName: result.user_lastName as string,
+            email: result.user_email as string,
+            id: result.user_id as number,
+            role: result.roleName as AllOrganizationRoles,
+            addedTime: result.createdTime as string,
+          });
+        });
+        if (getRequestId('searchUsers') === requestId) {
+          setResults(usersResults);
+        }
+      } else {
+        setResults(people);
+      }
```

Comment:
we are using a combination of real-time search results and cached results.
can we switch to using real-time results every time? just like what we discussed in other search related PRs

Reply:
Yes, I forgot about that. Will update it